src/test/run-pass/abi-sysv64-arg-passing.rs

```diff
@@ -98,7 +98,7 @@ mod tests {
     #[derive(Copy, Clone)]
     pub struct Floats { a: f64, b: u8, c: f64 }
 
-    #[link(name = "rust_test_helpers")]
+    #[link(name = "rust_test_helpers", kind = "static")]
```

Comment:
`rust_test_helpers` is a static lib, so it started failing on Windows after this change.  Granted it was only a couple of tests that used the `rust_dbg_static_mut` export, but I did a bulk change for consistency sake.

So yeah, there is some potential for breakage, but importing data from a library is a relatively rare thing, and it was already broken on Windows [half the time](https://github.com/dgrunwald/rust-cpython/issues/10#issuecomment-235976320).   Not sure how to assess the extent of this breake though.  There is no such thing as Windows crater, right?